cms/static/js/certificates/models/certificate.js

```diff
@@ -20,9 +20,25 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             name: 'Name of the certificate',
             description: 'Description of the certificate',
-            course_title: 'Title of the course',
+
+            /**
+             * It represents the course title.
+             */
+            course_title: '',
+
+            /**
+             * It represents the path to organization logo image.
+             */
             org_logo_path: '',
+
+            /**
+             * It represents the default version of the certificate.
```

Comment:
@ mattdrayer @asadiqbal08 I'm not sure what this "default version of certificate" is -- I don't see it in the UI?